editor/src/clj/editor/gl/vertex2.clj

```diff
@@ -451,11 +265,17 @@
 
 (defn- vertex-locate-attribs
   [^GL2 gl shader attribs]
-  (mapv #(shader/get-attrib-location shader gl (:name %)) attribs))
+  (let [attribute-infos (shader/attribute-infos shader gl)]
+    (mapv (fn [attrib]
+            (let [attribute-name (:name attrib)]
+              (if-some [attribute-info (get attribute-infos attribute-name)]
+                (:index attribute-info)
+                -1)))
+          attribs)))
```

Comment:
* Use the info we've already collected in `:attribute-infos` in functions that query shader info from the `ShaderLifecycle`.